main/pipeline/pipeline.cc

```diff
@@ -1254,74 +1254,6 @@ void printFileTable(unique_ptr<core::GlobalState> &gs, const options::Options &o
 #endif
 }
 
-bool cacheTreesAndFiles(const core::GlobalState &gs, WorkerPool &workers, absl::Span<const ast::ParsedFile> parsedFiles,
```

Comment:
You need to delete this from `pipeline.h` too

Reply:
thanks